src/utils/errors.ts

```diff
@@ -1,26 +1,11 @@
 import { StatusCodes } from 'http-status-codes';
 
-import { GraaspError, GraaspErrorDetails } from 'graasp';
+import { ErrorFactory } from '@graasp/sdk';
 
-export class GraaspImportZipError implements GraaspError {
-  name: string;
-  code: string;
-  message: string;
-  statusCode?: number;
-  data?: unknown;
-  origin: 'plugin' | string;
+import { PLUGIN_NAME } from '../constants';
 
-  constructor({ code, statusCode, message }: GraaspErrorDetails, data?: unknown) {
-    this.name = code;
-    this.code = code;
-    this.message = message;
-    this.statusCode = statusCode;
-    this.data = data;
-    this.origin = 'plugin';
-  }
-}
-
-export class FileIsInvalidArchiveError extends GraaspImportZipError {
+export const GraaspError = ErrorFactory(PLUGIN_NAME);
```

Comment:
IMO it still makes sense to have scoped error types:

```suggestion
export const GraaspItemZipError = ErrorFactory(PLUGIN_NAME);
```